### PR TITLE
Add dashboard creator as owner of the dashboard

### PR DESCRIPTION
**Pull request by bkyryliuk**
Adds creator as a owner of the dashboard.
Tested locally.

Issue:
https://github.com/airbnb/caravel/issues/780

Reviewers:
- @mistercrunch 
- @ascott 
- @vera-liu

### CODE REVIEW

File: caravel/views.py [hunk 1]
```diff
@@ -822,6 +822,7 @@ def pre_add(self, obj):
         if obj.slug:
             obj.slug = obj.slug.replace(" ", "-")
             obj.slug = re.sub(r'\W+', '', obj.slug)
+        obj.owners.append(g.user)
```
Comment (bkyryliuk) [inline caravel/views.py]: if creator puts himself into the owners - it will create 2 entries in the DB.

Comment (mistercrunch) [inline caravel/views.py]:
```
if g.user not in obj.owners:
    obj.owners.append(g.user)
```

**Comment by bkyryliuk** [comment]
@mistercrunch - done.

**Comment by mistercrunch** [comment]
Lgtm

**Pull request merged**